sagemaker-triton/fil_ensemble/2_triton_xgb_fil_ensemble.ipynb

```diff
@@ -0,0 +1,908 @@
+{
```

Comment:
<p>Recommended to have a Conclusion section summarizing what the customer accomplished, and any next steps for extending or adapting this example. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3524/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='63'/>

Reply:
Added Conclusion paragraph